### ISSUE TITLE
WASM failure on CI

### ISSUE THREAD

**Issue by itsankitkp** [RustPython/RustPython#4921]
I am getting following error on CI (with latest main)
```
Run wasmerio/setup-wasmer@v2
  with:
  env:
    CARGO_ARGS: --no-default-features --features stdlib,zlib,importlib,encodings,ssl,jit
    NON_WASM_PACKAGES: -p rustpython-common -p rustpython-compiler-core -p rustpython-compiler -p rustpython-codegen -p rustpython-parser -p rustpython-vm -p rustpython-stdlib -p rustpython-jit -p rustpython-derive -p rustpython
    PLATFORM_INDEPENDENT_TESTS: test_argparse test_array test_asyncgen test_binop test_bisect test_bool test_bytes test_call test_class test_cmath test_collections test_complex test_contains test_copy test_dataclasses test_decimal test_decorators test_defaultdict test_deque test_dict test_dictcomps test_dictviews test_dis test_enumerate test_exception_variations test_exceptions test_float test_format test_fractions test_genericalias test_genericclass test_grammar test_range test_index test_int test_int_literal test_isinstance test_iter test_iterlen test_itertools test_json test_keyword test_keywordonlyarg test_list test_long test_longexp test_math test_operator test_ordered_dict test_pow test_raise test_richcmp test_scope test_set test_slice test_sort test_string test_string_literals test_strtod test_structseq test_subclassinit test_super test_syntax test_tuple test_types test_unary test_unicode test_unpack test_weakref test_yield_from
    CARGO_INCREMENTAL: 0
    CARGO_TERM_COLOR: always
    CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
    CACHE_ON_FAILURE: false
Download the installer.
  (node:1767) UnhandledPromiseRejectionWarning: RequestError: Hostname/IP does not match certificate's altnames: Host: get.wasmer.io. is not in the cert's altnames: DNS:*.netlify.app, DNS:netlify.app
      at ClientRequest.<anonymous> (/home/runner/work/_actions/wasmerio/setup-wasmer/v2/dist/index.js:1:104464)
      at Object.onceWrapper (events.js:421:26)
      at ClientRequest.emit (events.js:326:22)
      at ClientRequest.e.emit (/home/runner/work/_actions/wasmerio/setup-wasmer/v2/dist/index.js:1:38646)
      at TLSSocket.socketErrorListener (_http_client.js:427:9)
      at TLSSocket.emit (events.js:314:20)
      at emitErrorNT (internal/streams/destroy.js:92:8)
      at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
      at processTicksAndRejections (internal/process/task_queues.js:84:21)
      at Object.checkServerIdentity (tls.js:287:12)
      at TLSSocket.onConnectSecure (_tls_wrap.js:1512:27)
      at TLSSocket.emit (events.js:314:20)
      at TLSSocket._finishInit (_tls_wrap.js:937:8)
      at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:711:12)
  (node:1767) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
  (node:1767) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

tldr: error is `RequestError: Hostname/IP does not match certificate's altnames: Host: get.wasmer.io. is not in the cert's altnames: DNS:*.netlify.app, DNS:netlify.app`

Refer: https://github.com/RustPython/RustPython/actions/runs/4766167600/jobs/8472864041

**Comment by itsankitkp** [comment]
Opened PR on setup-wasmer for this issue
https://github.com/wasmerio/setup-wasmer/issues/6

**Comment by itsankitkp** [comment]
This issue doesn't happen anymore.

**Comment by youknowone** [comment]
Thank you!
